packages/dds/tree/src/shared-tree/sharedTree.ts

```diff
@@ -171,7 +169,7 @@ class SharedTree
 	implements ISharedTree
 {
 	public readonly context: EditableTreeContext;
-	public readonly forest: IForestSubscription;
+	public readonly forest: IEditableForest;
```

Comment:
Note that this is exposing `IEditableForest` on `SharedTreeCore`, but not `ISharedTree`.